### ISSUE TITLE
duplicate results & deployment

### ISSUE THREAD

**Issue by Tigerold** [SmartBot-PJLab/P3Former#8]
Hi！
Thinks for your contributions for pointcloud panoptic segmentation! I want to duplicate the results using your code on SemanticKITTI dataset.
I have some question:
1. which config should I choose, form the three [configs ](https://github.com/OpenRobotLab/P3Former/tree/main/configs/p3former), and  "train_cfg"&"val_cfg"&"test_cfg" have a repeated definitions in config file.
2. how to set hyperparameters which is not clearly in the parper exprements parts and appendix to Implement baseline performence? 
3. do you release code on mmdetection3d platform?
4. the last, Can this model be  deployed on embedded platform? What do you have suggestion for the model convert engine file

Think you!!

**Comment by xizaoqu** [comment]
Hi.
1. p3former_8xb2_3x_semantickitti.py is for training on the training set and validating on the validation set.
p3former_8xb2_3x_semantickitti_trainval.py is for training on the training and validation set for better performance on the test set.
p3former_8xb2_3x_semantickitti_submmit is for getting the test results on the test set and submit online for evaluation.

2. Hyperparameters are included in the config files.

3.  The code is based on mmdetection3d. But we do not have a plan to incorporate it into mmdetection3d.

4.  I suppose various optimizations and modifications are required before it can be deployed on embedded platforms. We do not have a plan to do that yet.

**Comment by Tigerold** [comment]
Thinks for your reply.
